packages/backEnd/src/repositories/add.respositories.ts

```diff
@@ -55,6 +60,17 @@ export class Repository {
         })
         .toArray();
 
+      //   $where: function () {
```

Comment:
We shouldn't store unused code. If it is TODO, add this annotation.

Reply:
I will remove.